packages/react-core/src/components/DualListSelector/examples/DualListSelector.md

```diff
@@ -85,28 +85,50 @@ class BasicDualListSelectorWithSearch extends React.Component {
 
 ```js
 import React from 'react';
-import { Button, ButtonVariant, Dropdown, DropdownItem, DualListSelector, KebabToggle } from '@patternfly/react-core';
+import { Button, ButtonVariant, Dropdown, DropdownItem, DualListSelector, KebabToggle, Text, TextContent } from '@patternfly/react-core';
 import PficonSortCommonAscIcon from '@patternfly/react-icons/dist/js/icons/pficon-sort-common-asc-icon';
 
 class ComplexDualListSelector extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      availableOptions: [<span>Option 1</span>, <span>Option 3</span>, <span>Option 4</span>, <span>Option 2</span>],
+      availableOptions: [
+        <TextContent key={1}>
+          <span className="pf-c-dual-list-selector__item-text">Option 1</span>
```

Comment:
This element isn't really intended to be nested this way. If we don't want to build in a description prop for these elements for this kind of layout, we could pass this to the option instead

```
Option 1
<TextContent>
  <Text component={TextVariants.small}>Description 1</Text>
</TextContent>
```

That would generate this markup

<img width="362" alt="Screen Shot 2021-02-11 at 3 43 00 PM" src="https://user-images.githubusercontent.com/35148959/107702774-5b66a780-6c80-11eb-9b53-5b983958687d.png">

And it will also retain the component item's intended font-size, which you can see in the first option here, compared to other options that have `<TextContent>` wrapping all of the item text:

<img width="219" alt="Screen Shot 2021-02-11 at 3 43 04 PM" src="https://user-images.githubusercontent.com/35148959/107702861-7afdd000-6c80-11eb-94a7-af3b729bfc90.png">

Or if you need the `key` on something that wraps the whole item, just add a `<span key={...}>` around all of that.